Firestore/core/src/firebase/firestore/core/sync_engine.h

```diff
@@ -231,6 +240,16 @@ class SyncEngine : public remote::RemoteStoreCallback, public QueryEventSource {
 
   void TrackLimboChange(const LimboDocumentChange& limbo_change);
 
+  /**
```

Comment:
Ultranit: I understand this is copied from other platforms. I'd still suggest to mention max_concurrent_limbo_resolutions_ in the comment.